lib/modulr/api/transactions_service.rb

```diff
@@ -3,8 +3,8 @@
 module Modulr
   module API
     class TransactionsService < Service
-      def history(account_id:)
-        response = client.get("/accounts/#{account_id}/transactions")
+      def list(account_id:, **opts)
+        response = client.get("/accounts/#{account_id}/transactions", opts)
```

Comment:
I have a different approach to the parameters we pass to the API, specifically the naming. I'll comment on them in the PR for v0.0.7